01-builder-visualization/README.md

```diff
@@ -19,3 +19,8 @@ CARTO BUILDER for Data Visualization
 * Exercise: [category-bubbles](exercises/category-bubble.md)
```

Comment:
I don't like how they render I would use two lists of links with single headers, simpler and easy to read like you did on the day 2